modules/subscription/role_assignments.tf

```diff
@@ -10,4 +10,13 @@ resource "azurerm_role_assignment" "local_role_assignments" {
   scope                = each.value.scope
   role_definition_name = each.key
   principal_id         = each.value.principal_id
+
+}
+
+resource "azurerm_role_assignment" "local_custom_role_assignments" {
+  for_each           = local.custom_role_assignments
+  scope              = each.value.scope
+  role_definition_id = azurerm_role_definition.app_gateway_backend_health_reader.id
```

Comment:
this should be loaded from a resource or datasource I think not hardcoded to the role